app/models/application_record.rb

```diff
@@ -15,4 +15,35 @@ def update_sanitizer_version
   def self.random_order
     order(Arel.sql("RAND()"))
   end
+
+  def unmark_for_destruction
```

Comment:
Can you add a simple test that calls `mark_for_destruction` then `unmark_for_destruction` on a record, and expects `marked_for_destruction?` to be false?

Reply:
Just checking -- is this ready now that the tests have been added, or does it need a more thorough review?